src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj

```diff
@@ -13,6 +13,10 @@
     <Compile Include="Platforms\Uap\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard'))">
+    <Compile Include="Platforms\NetStandard\**\*.cs" />
```

Comment:
Why is this being included here and then the file being removed below?

Reply:
hmm think VS is doing some weird stuff, will take a look

Reply:
We should probably add this to most of the BuildIt csproj files